doc/user_guide/jupyter_chart.rst

```diff
@@ -0,0 +1,403 @@
+
+
+.. raw:: html
+
+    <video controls>
+      <source src="https://github.com/altair-viz/altair/assets/15064365/b8f70490-6a0b-4032-8ef8-95a21004b7ed">
+      Your browser does not support the video tag.
+    </video>
+
+
+Selection Params
+----------------
+JupyterChart makes it possible to access and observe selection parameters. For the purpose of accessing
+selections from Python, selection parameters are divided into three types:
+Point selections, index selections, and interval selection. These selection types are
+represented by Python classes named ``PointSelection``, ``IndexSelection``, and ``IntervalSelection``
+respectively.
+
+Instances of these selection classes are available as properties of the JupyterChart's
+``selections`` property.
+
+Point Selections
+~~~~~~~~~~~~~~~~
+The ``PointSelection`` class is used to store the current state of a Vega-Altair point selection
+(as created by ``alt.selection_point()``) when either a ``fields`` or ``encodings`` specification
+is provided. One common example is a point selection with ``encodings=["color"]`` that is bound to
+the legend.
+
+.. code-block:: python
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.cars()
+    brush = alt.selection_point(name="point", encodings=["color"], bind="legend")
+
+    chart = alt.Chart(source).mark_point().encode(
+        x='Horsepower:Q',
+        y='Miles_per_Gallon:Q',
+        color=alt.condition(brush, 'Origin:N', alt.value('grey')),
+    ).add_params(brush)
+
+    jchart = alt.JupyterChart(chart)
+    jchart
+
+.. raw:: html
+
+    <video controls>
+      <source src="https://github.com/altair-viz/altair/assets/15064365/9a48c9d7-ed84-4405-9125-8da58238f03b">
+      Your browser does not support the video tag.
+    </video>
+
+The ``PointSelection`` instance may be accessed as ``jchart.selections.point`` (Where "point" is the
+value of the ``name`` argument to ``alt.selection_point``).
+
+The ``jchart.selections.point.value`` property contains a list of dictionaries where each element
+represents a single point in the selection.
+
+Index Selections
+~~~~~~~~~~~~~~~~
+The ``IndexSelection`` class is used to store the current state of a Vega-Altair point selection
+(as created by ``alt.selection_point()``) when neither a ``fields`` nor ``encodings`` specification
+is provided.  In this case, the ``value`` property of the selection is a list of the indices
+of the selected rows. These indices can be used with the pandas DataFrame's ``iloc`` attribute to
+extract the selected rows in the input DataFrame.
+
+.. code-block:: python
+
+    import altair as alt
+    from vega_datasets import data
+
+    source = data.cars()
+    brush = alt.selection_point(name="point")
+
+    chart = alt.Chart(source).mark_point().encode(
+        x='Horsepower:Q',
+        y='Miles_per_Gallon:Q',
+        color=alt.condition(brush, 'Origin:N', alt.value('grey')),
+    ).add_params(brush)
+
+    widget = alt.ChartWidget(chart)
```

Comment:
This should read `JupyterChart`.

Reply:
Done in e097995767d635137f232735b67e4addde42294c